api_tests/lib/actor.js

```diff
@@ -45,6 +49,55 @@ class Actor {
                 });
         });
     }
+
+    async do(action) {
+        let network = action.payload.network;
+        if (network != "regtest") {
+            return Error("Expected network regtest, found " + network);
```

Comment:
❌ I think you're meant to throw Error in js: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error